src/editors/sharedComponents/TinyMceWidget/hooks.js

```diff
@@ -221,30 +255,14 @@ export const setupCustomBehavior = ({
   }
 
   editor.on('init', /* istanbul ignore next */ () => {
-    // Moving TinyMce aux modal inside the Editor modal
-    // if the editor is on modal mode.
-    // This is to avoid issues using the aux modal:
-    // * Avoid close aux modal when clicking the content inside.
-    // * When the user opens the `Edit Source Code` modal, this adds `data-focus-on-hidden`
-    //   to the TinyMce aux modal, making it unusable.
-    const modalLayer = document.querySelector('.pgn__modal-layer');
-    const tinymceAux = document.querySelector('.tox.tox-tinymce-aux');
-
-    if (modalLayer && tinymceAux) {
-      modalLayer.appendChild(tinymceAux);
+    if (editor.bodyElement?.closest('.pgn__modal')) {
+      // This editor is inside a Paragon modal. Use this hack to avoid interference with TinyMCE's own modal popups:
+      reparentTinyMceModals();
+      editor.on('selectionchange', reparentTinyMceModals);
     }
```

Comment:
The only editing issue I had here was when one of my problem's answers is an image:

https://github.com/user-attachments/assets/d6ba9079-4b3b-4bb3-b158-ab18b14931da

There's no way to select the embedded alt text to re-edit that image. But this was happening before, and is unrelated to this change.

Reply:
Weird. I wonder why it renders only the alt text and doesn't just show the (selectable) image. Would you mind opening that as a separate issue on this repo?

Reply:
Sure thing: https://github.com/openedx/frontend-app-authoring/issues/1654